### PR TITLE
convox scale shows desired and running process count

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;WUQq8R64&#x2F;199-convox-scale-shows-desired-actual-process-counts"> convox scale shows desired&#x2F;actual process counts</a>

File: api/models/formation.go [hunk 1]
```diff
@@ -134,3 +137,15 @@ func (f Formation) Entry(name string) *FormationEntry {
 
 	return nil
 }
+
```
Comment (csquared) [inline api/models/formation.go]: This is really cool -> define these functions and your struct is now sortable ?

**Comment by csquared** [comment]
:+1:

File: api/models/formation.go (hunk 1 above)
Comment (ddollar) [inline api/models/formation.go]: Yep https://golang.org/pkg/sort/#Interface

**Comment by mattmanning** [comment]
:+1:

**Pull request merged**